clients/sesNotifier.go

```diff
@@ -75,12 +77,31 @@ func NewSesNotifier(cfg *SesNotifierConfig) (*SesNotifier, error) {
 }
 
 // Send a message to a list of recipients with a given subject
-func (c *SesNotifier) Send(to []string, subject string, msg string) (int, string) {
+func (c *SesNotifier) Send(to []string, subject string, msg string, tags map[string]string) (int, string) {
 	var toAwsAddress = make([]*string, len(to))
 	for i, x := range to {
 		toAwsAddress[i] = aws.String(x)
 	}
+	confSetStr := c.Config.ConfigurationSet
+	var confSetName *string = nil
+	if confSetStr != "" {
+		confSetName = &confSetStr
+	}
 
+	allTags := make(map[string]string)
```

Comment:
I would put this block about tags preparation/merging in a separate methods just to keep the Send() method concise.